js/keybindings.js

```diff
@@ -0,0 +1,28 @@
+// var keyChar = require('./helpers/getKey');
+var momentChanger = require('./moments.js');
+
+module.exports = function() {
+  'use strict';
+  var unicode;
+
+  document.onkeydown = function(e) {
+    e.preventDefault();
+    unicode = e.keyCode ? e.keyCode : e.charCode;
+    console.log(unicode);
+
+    if(unicode === 32) {
+      console.log('space');
+      momentChanger.randomMoment(e);
```

Comment:
this. I'm pretty confused about how to this concept working (using randomMoment from momentChanger.js)